cli/cli_import.js

```diff
@@ -52,27 +50,6 @@ const cliUrl = '/cli';
 const cliNavFile = path.join('docs', 'nav.yml');
 const cliContentPath = path.join('docs', 'content');
 
-const indexMarkdown = `<Index depth="1" />`;
-
-const translations = {
```

Comment:
These titles are now generated directly from the cli nav: https://github.com/npm/cli/blob/latest/docs/nav.yml#L5